setup.py

```diff
@@ -46,15 +46,18 @@
 # compile with:     python setup.py build_ext -i
 # clean up with:    python setup.py clean --all
 if sys.platform == 'darwin':
-	extra_compiler_args = ['-stdlib=libc++']
+        # see https://github.com/pandas-dev/pandas/issues/23424
+	extra_compiler_args = ['-stdlib=libc++']  # not needed #, '-mmacosx-version-min=10.9']
```

Comment:
What's this # not needed part? Seems superfluous to our needs?

Reply:
per the pandas/numpy issue it should be needed, but appears not to be in my testing.